src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -135,6 +138,37 @@ data class MigratePunishment(
   @Schema(description = "days applied - note we do not support months, should be converted to days if present TBC")
   val days: Int?,
 )
+
+@Schema(description = "hearing and optional result")
+data class MigrateHearing(
+  @Schema(description = "oic hearing id")
+  val oicHearingId: Long,
+  @Schema(description = "oic hearing type")
+  val oicHearingType: OicHearingType,
+  @Schema(description = "date time of hearing")
+  val hearingDateTime: LocalDateTime,
+  @Schema(description = "hearing location id")
+  val locationId: Long,
+  @Schema(description = "adjudicator username - nomis has this on the hearing, so will accept at this level but for our service it goes on outcome")
+  val adjudicator: String?,
+  @Schema(description = "comment text")
+  val commentText: String?,
+  @Schema(description = "hearing result", required = false)
+  val hearingResult: MigrateHearingResult? = null,
```

Comment:
assumption being made here is that for a single charge sequence, there can only be one result  line per hearing?  May need to run some SQL on prod to confirm this.